diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/EnumsSeparatedWarnTest.kt

```diff
@@ -0,0 +1,181 @@
+package org.cqfn.diktat.ruleset.chapter3
+
+import com.pinterest.ktlint.core.LintError
+import generated.WarningNames
+import org.cqfn.diktat.ruleset.constants.Warnings.ENUMS_SEPARATED
+import org.cqfn.diktat.ruleset.rules.DIKTAT_RULE_SET_ID
+import org.cqfn.diktat.ruleset.rules.EnumsSeparated
+import org.cqfn.diktat.util.lintMethod
+import org.junit.jupiter.api.Tag
+import org.junit.jupiter.api.Test
+
+class EnumsSeparatedWarnTest {
+
+    private val ruleId = "$DIKTAT_RULE_SET_ID:enum-separated"
+
+    @Test
+    @Tag(WarningNames.ENUMS_SEPARATED)
+    fun `check simple correct enum with new line`() {
+        lintMethod(EnumsSeparated(),
+                """
+                    |enum class ENUM {
```

Comment:
Please also add example with comments on each line, for example this should be valid:
```kotlin
enum class ENUM {
   A, // this is A
   B,
   ;
```